resource/doc/BuildForWindows.md

```diff
@@ -1,8 +1,39 @@
-# Prepare  [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/68i57b8ssasttngg?svg=true)](https://ci.appveyor.com/project/li-zhi/vmaf)
-  - Visual Studio 2015 on Windows
+# Building vmaf on Windows
+
+We are going to build vmaf using powershell. These steps are in accordance with the corresponding github action for building on Windows and have been tested successfully using a Windows10 machine.
+
+### Requirements
```

Comment:
nit: I do not recommend going from `H1` (the `#`) straight to `H3` (`###`) and instead descend incrementally to `##`